src/Stratis.Bitcoin.Features.SignalR/Events/StakingInfoClientEvent.cs

```diff
@@ -0,0 +1,45 @@
+namespace Stratis.Bitcoin.Features.SignalR.Events
+{
+    using System;
```

Comment:
`using`s should be at the top of the file.

Reply:
Done